modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java

```diff
@@ -309,6 +334,39 @@ private JSONObject _getClipboardJSONObject(
 			});
 	}
 
+	private boolean _isSharingEnabled(
+		long classNameId, long classPK, ContentDashboardItem<?> contentDashboardItem,
+		ThemeDisplay themeDisplay ) {
+
+		SharingConfiguration sharingConfiguration =
+				_sharingConfigurationFactory.getGroupSharingConfiguration(
+				themeDisplay.getSiteGroup());
+
+		if (!sharingConfiguration.isEnabled()) {
+			return false;
+		}
+
+		boolean sharingEnabled = false;
+
+		try {
+			sharingEnabled = _sharingPermission.containsSharePermission(
```

Comment:
return fast, instead save the result in a variable.

Reply:
Done! thank you!